### PR TITLE
Update third-party chart versions

### PR DESCRIPTION
**Pull request by NicolasT**
- Basic test for `metrics-server`
- Updates of various chart versions
- Update Elasticsearch to use `RollingUpdate` deployments

### CODE REVIEW

**Comment by NicolasT** [comment]
To be investigated why the `metrics-server` test is failing.

**Comment by NicolasT** [comment]
Tests were failing due to https://github.com/scality/metalk8s/pull/373/files#diff-36d18a34df637e8f16174465ed8e3350R8 not being there (DNS resolution of `ci-metalk8s` fails, of course).

**Comment by NicolasT** [comment]
Ugh, new failure. Potentially timing issue: metrics-server doesn't immediately provide metrics, only after some time...

**Comment by NicolasT** [comment]
Should be fixed... https://github.com/scality/metalk8s/pull/373/commits/69c8125afcc93fd82954d9b41188be21bc91ce55#diff-e7316cf09e38171b2f945d5fa5f92032R5

**Review by anurag4DSB — APPROVED** [review]

File: roles/kube_metrics_server/vars/main.yml [hunk 1]
```diff
@@ -2,6 +2,9 @@
 kube_metrics_server__default_values:
   args:
   # Default:
-  #- --source=kubernetes.summary_api:''
-  # For https://github.com/scality/metalk8s/issues/36:
-  - "--source=kubernetes.summary_api:''?useServiceAccount=true&kubeletHttps=true&kubeletPort=10250&insecure=true"
+  - --logtostderr
+  # MetalK8s-specific:
+  - --kubelet-insecure-tls
+  # Required to make sure node hostnames aren't resolved through DNS
```
Comment (Zempashi) [inline roles/kube_metrics_server/vars/main.yml:8]: Should we adapt the inventory to not change the hostname of the server and get DNS working ?

Comment (NicolasT) [reply roles/kube_metrics_server/vars/main.yml:8]: Then you rely on an external DNS server to resolve the hostname, which is not necessarily the case. Defaulting to InternalIP should be fine in most deployments, no? In the end, that's the IP `kubelet`s are reporting themselves.

Comment (Zempashi) [reply roles/kube_metrics_server/vars/main.yml:8]: Yes, indeed. Now wondering why it's not the default

Comment (Zempashi) [reply roles/kube_metrics_server/vars/main.yml:8]: maybe for tls...

Comment (NicolasT) [reply roles/kube_metrics_server/vars/main.yml:8]: Kubespray not generating certificates correctly is why we need `--kubelet-insecure-tls` in the first place :-(

**Pull request merged**